topics/journalpost.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: journalpost
+  namespace: bidrag
+  labels:
+    team: bidrag
+spec:
+  pool: {{kafka-pool}}
+  config:
+    cleanupPolicy: compact
+    minimumInSyncReplicas: 1
+    partitions: 1
+    replication: 3
+    retentionBytes: -1
+    retentionHours: 72
```

Comment:
Vi snakket vel om at retention kunne være evig for journalpost. Alternativt bør den vel være mer enn 3 dager, slik at vi kan følge opp eventuelle feilsituasjoner.

Reply:
stemmer... jeg la ut et spørsmål på #kafka som aldri har blitt besvart... dette gikk derfor i glemmeboka... jeg skal undersøke hva den kan inneholde?